docs/pages/reference/access-controls/roles.mdx

```diff
@@ -120,7 +120,7 @@ Let's say you create an intern role with the allow rule for label `environment=s
 ### Example
 
 The role below allows access to all nodes labeled "env=stage" except those that
-also have "workload=database" or "workload=backup".
+also have "workload=database" or "workload=backup". A global idle timeout of 30 minutes is set.
```

Comment:
I think it'd make sense to swap "A global idle timeout of 30 minutes is set" for "Access to any other nodes will be denied", since the sentence re: the idle timeout comes between two sentences about labels.